### DISCUSSION TITLE
2011/02/27/%E8%B6%8A%E6%9D%A5%E8%B6%8A%E4%B8%8D%E5%96%9C%E6%AC%A2%E8%B0%B7%E6%AD%8C%E6%8B%BC%E9%9F%B3%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#314] [Announcements]
# 2011/02/27/%E8%B6%8A%E6%9D%A5%E8%B6%8A%E4%B8%8D%E5%96%9C%E6%AC%A2%E8%B0%B7%E6%AD%8C%E6%8B%BC%E9%9F%B3%E4%BA%86

我本来是Google的忠实用户，输入法也一直在用谷歌的。可是自打Google撤离中国，谷歌的拼音输入法就再没有过什么重大的改进了。前一段时间发现它出了3.0版，心里大喜，赶快下来试用。

https://blog.qizhen.xyz/2011/02/27/%E8%B6%8A%E6%9D%A5%E8%B6%8A%E4%B8%8D%E5%96%9C%E6%AC%A2%E8%B0%B7%E6%AD%8C%E6%8B%BC%E9%9F%B3%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

ruanqizhen
2011年03月27日 14:30 编辑
不要紧:)

回复
mangguopai
2011年03月26日 19:13 编辑
呵呵，转载你的贴到谷拼的论坛，实现没告知你哦。

回复
ijayci
2011年02月28日 15:32 编辑
搜狗输入法没广告啊, 很好用

回复
sibylfan
2011年02月28日 13:30 编辑
搜狗输入法不会弹广告呀
最多第一次用的时候会弹一个小窗口，显示些新闻、图片什么的，但这个也可以不让它显示的

回复
勺子
2011年02月28日 12:35 编辑
勺子也算GOOGLE迷吧，用MAP，GMAIL，READER。。。不一而足
但这古狗输入法倒是从来木有用过的了。。。

回复
勺子
2011年02月28日 12:35 编辑
一直用MAC自带的输入法，感觉不错用哈。

回复
Yang
2011年02月28日 12:13 编辑
多年前有一个输入法叫拼音加加。后来实在是竞争不过大公司的宣传攻势，市场占有率没了。前不久它还更新了一下。可以试试看。

回复
